### DISCUSSION TITLE
I can't let my friend join my grasscutter server(我无法让我的好友加入服务器)

### DISCUSSION THREAD

**Discussion by Miaryi** [Melledy/Grasscutter#1550] [Q&A]
EN
Hello，I have a question and ask for help
i can‘t let my friend join in my grasscutter server

The game version is 2.8.51_beta i use
![image](https://user-images.githubusercontent.com/72502875/180473519-06bd5e5c-16d1-4da7-bd37-269fd6d3ed5b.png)

I have successfully set up a grasscutter server.
i wanna my friend join the server
so I try to change the configuration of my packet capture tool (fidder) ,but it doesn't work.
![image](https://user-images.githubusercontent.com/72502875/180473639-82d93a73-64df-41d1-82ae-bcce4cd082ec.png)

Always Connection timed out.
the Error code：4206
![image](https://user-images.githubusercontent.com/72502875/180475833-2c679bfa-b6c2-419e-b202-019327abef2f.png)

What shall I do? Are there any related tutorials i can read?
CH
你好，我有一个问题
服务器没法联机
我正在使用的客户端版本是2.8.51
我已经架设好服务器，拉我朋友进来做角色评测
我用的抓包工具是fidder，尝试修改配置，但好像不行
我自己测试的，碰到的问题是网络超时，但是我这里认识的一个群友，他发过来的是4206错误（不是上面提到的人，他是个整合包author）
我这里用的是Sakura frp，本机端口 address127.0.0.1  |  host22102，网络映射后台没有任何回应
有教程course之类吗
如果我是加入服务器的玩家，我该怎么做，进入服务器？

**Comment by dianwangwin** [comment]
不处理关于某动画游戏公司Beta版本的issues

**Reply by Miaryi** [reply]
这个不是重点，我想问如何进行联机connection（请忽略版本问题的部分

**Reply by Miaryi** [reply]
请无视掉版本部分，请扔给我一个course link看也可
